index.js

```diff
@@ -69,4 +74,61 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const button = document.querySelector('button.like-button')
+// console.log(button)
+
+button.addEventListener('click', function(){
+const likesPtag = document.querySelector('p.likes')
+const currLikes = parseInt(likes.textContent)
+
+
+likes.textContent = `${currLikes + 1} Likes`
+
+// likes.textContent = `${traveler.likes++} Likes`
+
+})
+
 /***** Deliverable 3 *****/
+
+const newSightingForm = document.querySelector('form#new-animal-sighting-form')
+
+newSightingForm.addEventListener('submit', function(event){
+    event.preventDefault()
+    //console.log(event.target)
+
+
+    const species = event.target[0].value 
+    const video= event.target[1].value
+    const photo = event.target[2].value
+    const description = event.target[3].value
+
+    //another way - using the id or name attribute on the element
+    // you can use dot notation if the value doesn't have any special characters 
+    // const species = event.target.species.value 
+    // const video= event.target["animal-link"].value
+    // const photo = event.target["animal-photo"].value
+    // const description = event.target["sighting-description"].value
+ 
+    // get the user input 
+
+
+const lastIndex = traveler.animalSightings.length - 1
+const newSightingObject = {
+    id: traveler.animalSightings[lastIndex].id + 1 ,
+    travelerId: 1,   //hard coded 
+    species: species,
+    photo: photo,
+    link:  video,
+    description: description
+}
+
```

Comment:
Here, we could include:
`traveler.animalSightings.push(newSightingObject)`
It is a good idea to include this line because of the way we're creating custom IDs. If we want future animal sightings to have a unique ID, then we have to push all new animal sighting objects into the array.